app_usb_aud_xk_316_mc/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(SW_USB_AUDIO_FLAGS ${EXTRA_BUILD_FLAGS} -O3
                                             -g
                                             -fxscope
                                             -DBOARD_SUPPORT_BOARD=XK_AUDIO_316_MC_AB
+                                            -DXUA_DESC_INPUT_TYPE_LINE_IN
```

Comment:
should be = 1 (see comment in lib_xua PR)

we should also add a changelog entry